pkg/nuget/packages_props/parse.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/xml"
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+	
+
+
+type propsPackageEntry struct{
+	Version         string   `xml:"Version,attr"`
+	UpdatePackageName     string   `xml:"Update,attr"`
+	IncludePackageName     string   `xml:"Include,attr"`
+}
+
+type propsPackageReferenceEntry struct {
+	XMLName         xml.Name `xml:"PackageReference"`
+	propsPackageEntry
+}
+
+type propsPackageVersionEntry struct {
+	XMLName         xml.Name `xml:"PackageVersion"`
+	propsPackageEntry
+}
+
+type propsItemGroup struct {
+	XMLName  xml.Name              `xml:"ItemGroup"`
+	ReferencePackages []propsPackageReferenceEntry `xml:"PackageReference"`
+	VersionPackages []propsPackageVersionEntry `xml:"PackageVersion"`
+}
+
+type propsProject struct {
+	XMLName xml.Name `xml:"Project"`
+	ItemGroups []propsItemGroup `xml:"ItemGroup"`
+}
+
+type Parser struct{}
+
+func NewParser() types.Parser {
+	return &Parser{}
+}
+
+func (p *Parser) addPackage(libs* []types.Library, pkg propsPackageEntry) {
```

Comment:
You save libs not explicitly 
This is not good and can be confusing.

Better to explicitly return lib.
```suggestion
func parsePackage(pkg propsPackageEntry) types.Library {
```

Reply:
Sorry, I didn't think about that when I wrote this comment:
What if we just use `Library` name?
```
func library(pkg entry) types.Library {
```